backend/dal/Migrations/Initial/Up/PostUp/01-Agencies.sql

```diff
@@ -1,556 +1,88 @@
-), (
-    14
-    , 'SDPR'
-    , 'Ministry of of Social Development and Poverty Reduction'
-    , 0
-    , 1
-    , 'jonathan.dube@gov.bc.ca'
-    , 'ADM & EFO CSD'
-), (
-    15
-    , 'MMHA'
-    , 'Ministry of Mental Health and Addictions'
-    , 0
-    , 1
-    , 'dara.landry@gov.bc.ca'
-    , 'Executive Lead Corporate Services'
-), (
-    16
-    , 'MCFD'
-    , 'Ministry of Children and Family Development'
-    , 0
-    , 1
-    , 'rob.byers@gov.bc.ca'
-    , 'AMD & EFO'
-), (
-    17
-    , 'BCPSA'
-    , 'Public Service Agency'
-    , 0
-    , 1
-    , 'bruce.richmond@gov.bc.ca'
-    , 'ADM Corporate Services'
-)
-
--- Child Agencies for HLTH.
-INSERT INTO dbo.[Agencies] (
-    [Id]
-    , [ParentId]
-    , [Code]
-    , [Name]
-    , [IsDisabled]
-    , [SendEmail]
-) VALUES (
-    20
-    , 7
-    , 'FHA'
-    , 'Fraser Health Authority'
-    , 0
-    , 0
-), (
-    21
-    , 7
-    , 'IHA'
-    , 'Interior Health Authority'
-    , 0
-    , 0
-), (
-    22
-    , 7
-    , 'NHA'
-    , 'Northern Health Authority'
-    , 0
-    , 0
-), (
-    23
-    , 7
-    , 'PHSA'
-    , 'Provincial Health Services Authority'
-    , 0
-    , 0
-), (
-    24
-    , 7
-    , 'VCHA'
-    , 'Vancouver Coastal Health Authority'
-    , 0
-    , 0
-), (
-    25
-    , 7
-    , 'VIHA'
-    , 'Vancouver Island Health Authority'
-    , 0
-    , 0
-)
-
--- Child Agencies for EMPR.
-INSERT INTO dbo.[Agencies] (
-    [Id]
-    , [ParentId]
-    , [Code]
-    , [Name]
-    , [IsDisabled]
-    , [SendEmail]
-) VALUES (
-    30
-    , 10
-    , 'BCH'
-    , 'BC Hydro'
-    , 0
-    , 0
-)
-
--- Child Agencies for MAH.
-INSERT INTO dbo.[Agencies] (
-    [Id]
-    , [ParentId]
-    , [Code]
-    , [Name]
-    , [IsDisabled]
-    , [SendEmail]
-) VALUES (
-    40
-    , 8
-    , 'BCH'
-    , 'BC Housing'
-    , 0
-    , 0
-), (
-    41
-    , 8
-    , 'BCA'
-    , 'BC Assessment'
-    , 0
-    , 0
-), (
-    42 -- Added after release.
-    , 8
-    , 'PRHC'
-    , 'Provincial Rental Housing Corporation'
-    , 0
-    , 0
-)
-
--- Child Agencies for MAG.
-INSERT INTO dbo.[Agencies] (
-    [Id]
-    , [ParentId]
-    , [Code]
-    , [Name]
-    , [IsDisabled]
-    , [SendEmail]
-) VALUES (
-    50
-    , 11
-    , 'ICBC'
-    , 'Insurance Coporation of BC'
-    , 0
-    , 0
-),(
-    51 -- Added after release.
-    , 11
-    , 'LDB'
-    , 'BC Liquor Distribution Branch'
-    , 0
-    , 0
-)
-
--- Child Agencies for JEDC.
---INSERT INTO dbo.[Agencies] (
---    [Id]
---    , [ParentId]
---    , [Code]
---    , [Name]
---    , [IsDisabled]
---    , [SendEmail]
---) VALUES (
---    50
---    , 12
---    , 'BCPC'
---    , 'BC Pavillion Corporation'
---    , 0
---    , 0
---)
-
--- Child Agencies for EDUC.
-INSERT INTO dbo.[Agencies] (
-    [Id]
-    , [ParentId]
-    , [Code]
-    , [Name]
-    , [IsDisabled]
-    , [SendEmail]
-) VALUES (
-    70
-    , 4
-    , 'CMB'
-    , 'Capital Management Branch'
-    , 0
-    , 0
+MERGE INTO dbo.[PIMS_AGENCY] dest
```

Comment:
why are some of these straight inserts and some merges? isn't everything here Initial and therefore only an insert is required?

Reply:
I think in this case I was just copy+paste and kept it in.